x-pack/test/api_integration/apis/ingest/index.js

```diff
@@ -0,0 +1,11 @@
+/*
```

Comment:
I think we need the integrations tests to show the limited features we added work as expected.

Reply:
Agree with this sentiment; the only test suite that is currently being loaded is skipped, so if we want to re-enable it, we'll need to modify it so that it runs correctly